README.md

```diff
@@ -49,6 +48,60 @@ def example_s3_upload_flow():
 example_s3_upload_flow()
 ```
 
+### Write and run a flow with AwsCredentials and S3Bucket
+
+```python
+import asyncio
+from prefect import flow
+from prefect_aws import AwsCredentials, S3Bucket
+
+@flow
+async def aws_s3_bucket_roundtrip(creds):
+
+    s3_bucket = S3Bucket(
+        bucket_name="bucket",  # must exist
+        aws_credentials=creds,
+        basepath="subfolder",
+    )
+
+    key = await s3_bucket.write_path("data.csv", content=b"hello")
+
+    return await s3_bucket.read_path(key)
+
+# create an AwsCredentials block here or through UI
+aws_creds = AwsCredentials(
+    aws_access_key_id=AWS_ACCESS_KEY_ID,
+    aws_secret_access_key=AWS_SECRET_ACCESS_KEY
+)
+
+asyncio.run(aws_s3_bucket_roundtrip(aws_creds))
+```
+
+### Write and run an async flow by loading a MinIOCredentials block to use in S3Bucket
+
+```python
+import asyncio
+from prefect import flow
+from prefect_aws import MinIOCredentials, S3Bucket
```

Comment:
`from prefect_aws.s3 import S3Bucket`

Reply:
[Update examples in readme](https://github.com/PrefectHQ/prefect-aws/pull/47/commits/a56695de2dee9d2118c6b2a66e42f44cd030a174)